migrate.py

```diff
@@ -357,8 +358,12 @@ def rewrite_unit_tests_patch(mod_fst, collection, spec, namespace, args, filenam
             elif val[1] == 'plugins':
                 # 'ansible.plugins.lookup.manifold.open_url'
                 plugin_type = val[2]
-                plugin_name = val[3]
-
+                try:
+                    plugin_name = val[3]
+                except IndexError as e:
+                    logger.warn('%s %s' % (el, val))
```

Comment:
It's strongly recommended to pass vars via logging func arguments instead of interpolating strings manually:
```suggestion
                    logger.warn('%s %s', el, val)
```